docs/quickstart/index.rst

```diff
@@ -0,0 +1,22 @@
+Quickstart Guide
+==================
+
+.. _MetalK8s: https://github.com/scality/metalk8s/
+
+This guide describes how to set up a MetalK8s_ cluster to play with. It
+offers general requirements and describes sizing, configuration, and
+deployment. It also explains major concepts central to MetalK8s architecture,
+and will show how to access various services after completing the setup.
+
+With respect to installation procedures, the only significant difference
```

Comment:
Yeah, just an excerpt from the previous Quickstart, I'll rephrase (though the notion of hardware recommendation for production architectures is not that crucial for such a guide IMO).

Reply:
Actually, I'll remove this paragraph, since the Quickstart is designed for building a test cluster, not a production one. Prod concerns will come with the extended `Installation guide`, which I refer to above.